src/test/java/picard/analysis/CollectOxoGMetricsTest.java

```diff
@@ -12,7 +12,7 @@
 
 public class CollectOxoGMetricsTest {
     private static final File TEST_DATA_DIR = new File("testdata/picard/sam/");
-    private static final File SAM_FILE = new File(TEST_DATA_DIR, "summary_alignment_stats_test.sam");
+    private static final File SAM_FILE = new File(CollectAlignmentSummaryMetricsTest.TEST_DATA_DIR, "summary_alignment_stats_test.sam");
```

Comment:
This seems like it could lead to problems, if someone changed the file while editing the other test but didn't realize the file was used here as well. If you made a separate constant for each file shared like this (e.g., build the path to the file in `CollectAlignmentSummaryMetricsTest` as a constant), that would at least make it easier to find other tests that depended on that file.

Reply:
This is no different than what it was (regarding someone changing the file) but at least now there's a variable that _is_ traced to where it's being used...I fail to see how building the path from scratch helps...I moved the files into a subdirectory and suddenly other tests started failing....with this construction it will not happen...